packages/account-v2/src/utils/formattedLoginHistoryUtils.ts

```diff
@@ -0,0 +1,28 @@
+import dayjs from 'dayjs';
+import UAParser from 'ua-parser-js';
+import { LoginHistory } from '@deriv/api-types';
+
+export const formattedLoginHistoryUtils = (loginHistory: LoginHistory) =>
```

Comment:
maybe we can come up with better name for this function? it says it returns utils, but actually it is the array of objects, how do you think?